get_cover_art/meta_vorbis.py

```diff
@@ -0,0 +1,41 @@
+from .meta_audio import MetaAudio
+# It might look odd to be importing from FLAC in the opus
+# metadata encoder, but the VorbisComment spec (used by OGG Vorbis)
+# specifies that # it uses the FLAC structure, base64-encoded.
```

Comment:
This comment mentions "opus" in meta_vorbis.py- I think it should say "vorbis" instead.

Also, line 4 has an extra # character mid-line